src/SwedbankPay.Sdk/Payments/TrustlyPayments/TrustlyPaymentRequest.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+
+namespace SwedbankPay.Sdk.Payments.TrustlyPayments
+{
+    public class TrustlyPaymentRequest
+    {
+        public TrustlyPaymentRequest(
+            CurrencyCode currency,
+            List<Price> prices,
+            string description,
+            string payerReference,
+            string userAgent,
+            CultureInfo language,
+            Urls urls,
+            PayeeInfo payeeInfo) : this(currency, prices, description, payerReference, userAgent, language, urls, payeeInfo, null)
+        {
+        }
+
+
+        public TrustlyPaymentRequest(
+            CurrencyCode currency,
+            List<Price> prices,
+            string description,
+            string payerReference,
+            string userAgent,
+            CultureInfo language,
+            Urls urls,
+            PayeeInfo payeeInfo,
+            PrefillInfo prefillInfo)
+        {
+            Payment = new PaymentRequestObject(currency, prices, description, payerReference, userAgent, language,
+                                               urls, payeeInfo, prefillInfo);
+        }
+
+
+        public PaymentRequestObject Payment { get; }
+    }
+
+    public class PaymentRequestObject
+    {
+        protected internal PaymentRequestObject(CurrencyCode currency,
+                                                List<Price> prices,
+                                                string description,
+                                                string payerReference,
+                                                string userAgent,
+                                                CultureInfo language,
+                                                Urls urls,
+                                                PayeeInfo payeeInfo,
+                                                PrefillInfo prefillInfo = null)
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5814474)